src/renderer/stores/accountNotifications.ts

```diff
@@ -0,0 +1,82 @@
+/// this store is to keep track of what accounts are muted, basically for caching the core ui config key.
+/// in the future it can also cache custom notifcations sounds
```

Comment:
Why is caching necessary? Does it give us any performance benefits we can measure?

Reply:
it removes needless roundtrips to core and allows us to synchronously check the fields.

Reply:
Caching is an optimization and I'd be cautious about introducing that too early, especially with it being hard to debug and to control.

To me it currently sounds like we would also be fine without it

Reply:
> it also gives us a clean & easy way make sure all react components showing this information are updated when we update it.

That would be the role of a state handling logic but not of a caching layer

Reply:
> Caching is an optimization and I'd be cautious about introducing that too early, especially with it being hard to debug and to control.

While I agree with your statement in general, I think it is the right solution in this case.

Reply:
Thanks for clarifying, I would like to look at this closer some time!